kevlar/include/kevlar_bits/driver/accumulate.hpp

```diff
@@ -40,17 +40,15 @@ inline void accumulate(const SGSType& sgs, const GridRangeType& grid_range,
     }
 
     if (n_threads > max_threads) {
-        n_threads %= max_threads;
+        n_threads = max_threads;
     }
 
     auto sim_size_thr = sim_size / n_threads;
     auto sim_size_rem = sim_size % n_threads;
 
     std::vector<acc_t> acc_os(n_threads, acc_o);
 
-    omp_set_num_threads(n_threads);
-
-#pragma omp parallel for schedule(static)  // TODO: add some args
+#pragma omp parallel for schedule(static) num_threads(n_threads)
```

Comment:
This sets the number of threads in a less global way. Not sure if this is preferable, but I just made the change since I saw it.